pkg/util/azureclient/authz/remotepdp/client.go

```diff
@@ -87,6 +88,6 @@ func newCheckAccessError(r *http.Response) error {
 	return &azcore.ResponseError{
 		StatusCode:  r.StatusCode,
 		RawResponse: r,
-		ErrorCode:   checkAccessError.Message,
+		ErrorCode:   fmt.Sprint(checkAccessError.StatusCode),
```

Comment:
is this intentional?  Changing errorcode to point to the status code instead of the message on the error?

Reply:
Hmm I was kinda in two minds about this.  In the end, I made this change because according the description of ResponseError field shown below.  I assume the code might be used somewhere like being returned from cli, and it should be a code not a message.  The raw response is still included in the error so the message is not actually lost.

![image](https://github.com/Azure/ARO-RP/assets/9658731/9f7891e3-f3cc-4a76-9bff-3e57ce22247d)

But let me know what you think. I can change it back if returning the message is beter.

Reply:
I tend to agree with Nont on this. The documentation says code. I would try it and see if it is bad or not.